Jenkinsfile

```diff
@@ -86,16 +86,70 @@ try {
         }
     }
 
+    tasks['runPylint']  {
+        stage('runPylint')
+        {
+            try {
```

Comment:
This is not running in a node context. Please add node(....) and withContainer ....